src/main/java/com/faforever/api/utils/Validator.java

```diff
@@ -16,10 +18,8 @@ private Validator() {
    * @param object the object to check for null
    * @param message the exception message
    */
-  public static void notNull(Object object, String message) {
-    if (object == null) {
-      throw new NullPointerException(message);
-    }
+  static void notNull(Object object, String message) {
```

Comment:
The idea was to remove this method completely, we don't need a static method to wrap a static method :) (and btw. you're ignoring `message`)